corpus/arguments.txt

```diff
@@ -152,3 +152,69 @@ EXTERNAL *netrw-externapp* {{{2
         (tag
           (word))
         (word)))))
+
+================================================================================
+Many arguments
+================================================================================
+
+vim.foo({bar})
+
+vim.foo( {bar})
+
+nvim_foo({bar})
+
+nvim_foo({bar},{baz})
+
+nvim_foo({bar}, {baz})
+
+nvim_buf_detach_event[{buf}]
```

Comment:
args may have whitespace (`{arg arg}`) but that's already tested on line 4 so if that didn't regress, 👍